src/ducks/Signals/signalFormManager/signalCrudForm/signal/TriggerForm.js

```diff
@@ -115,10 +115,7 @@ export const TriggerForm = ({
   )
 
   const {
-    settings: {
-      isTelegramConnectedAndEnabled: isTelegramConnected,
-      isEmailConnected
-    }
+    settings: { isTelegramAllowAlerts: isTelegramConnected, isEmailConnected }
```

Comment:
`isEmailConnected` or here should be `isEmailAllowAlerts` ?

Reply:
Thx, bug